File: Sakila_Net/Sakila_Net/VB_SakilaLib/DAO/DAOStaff.vb
'using System;
Imports  System.Data
'using System.Collections.Generic;
'using System.Configuration;
'using System.Collections;
Imports  SakilaLib.Cl.Harrys.Sakila.Model
'using Cl.Harrys.BD;

Namespace Cl
    Namespace Harrys
        Namespace Sakila
            Namespace DAO

                Public Class DAOStaff
                    'Inherits SakilaLib.Cl.Harrys.Sakila.DAOBD.DAOStaff
                    Public Shared  Function SelectAll(nPagina as Integer ,cantidad as Integer) as DataTable
                        Return DAOBD.DAOStaff.SelectAll(nPagina, cantidad)	   
                    End Function
                    Public Shared  Function SelectOne( staff as ClStaff) as Boolean
                        Return DAOBD.DAOStaff.SelectOne(staff)     
                    End Function
                    Public Shared  Function SqlInsert( staff as ClStaff) as Integer
                        Return DAOBD.DAOStaff.SqlInsert(staff)     
                    End Function
                    Public Shared  Function SqlDelete( staff as ClStaff) as Integer
                        Return DAOBD.DAOStaff.SqlDelete(staff)     
                    End Function
                    Public Shared  Function SqlUpdate( staff as ClStaff) as Integer
                        Return DAOBD.DAOStaff.SqlUpdate(staff)     
                    End Function
                End Class

            End Namespace  
        End Namespace  
    End Namespace  
End Namespace  

File: Sakila_Net/CS_SakilaLib/DAO/DAOStaff.cs
//using System;
using System.Data;
//using System.Collections.Generic;
//using System.Configuration;
//using System.Collections;
using Cl.Harrys.Sakila.Model;
namespace  Cl.Harrys.Sakila.DAO
{   
  public class DAOStaff 
  {
       public static DataTable sqlSelectAll(int nPagina,int cantidad)
	   {
           return DAOBD.DAOStaff.selectAll(nPagina,cantidad);
	   }
       public static bool sqlSelect(ClStaff staff)
	   {
            return DAOBD.DAOStaff.SelectOne(staff); 
	   }
	   public static int sqlInsert(ClStaff staff)
	   {
            return DAOBD.DAOStaff.sqlInsert(staff);
       }
	   public static int sqlDelete(ClStaff staff)
	   {
            return DAOBD.DAOStaff.sqlDelete(staff);
       }
	   public static int sqlUpdate(ClStaff staff)
	   {
            return DAOBD.DAOStaff.sqlUpdate(staff);
       }
   }
}
